.github/workflows/reusable_build_packages.yaml

```diff
@@ -38,6 +46,16 @@ jobs:
       - name: Install Rust 🦀
         uses: dtolnay/rust-toolchain@f9f65f52341ba3c1d5e1901c77dc7a9e58186191 # stable
 
+      - name: Install Rust (nightly) 🦀
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: "nightly-2025-04-27"
+          components: "rust-src"
```

Comment:
Which is the correct one? :D 
or do we need 2 rust toolchains?

Reply:
We need two toolchains :sob:

Reply:
:skull: